### ISSUE TITLE
Some districts have too many active legislators.

### ISSUE THREAD

**Issue by estaub** [openstates/openstates#1389]
I believe these districts have more active legislators (from the legislator API) than there are seats in the district (from the district API).  I've spot-checked them, but of course it's possible that my thinking and/or code is buggy.  This might be an area where more scraper validation may be in order.  If you want more info output here, let me know what's needed.  You may want to reproduce the result anyway.

```
ar: District ar-lower-60 has 1 too many active legislators
ar: District ar-lower-81 has 1 too many active legislators
ar: District ar-lower-38 has 1 too many active legislators
ar: District ar-lower-1 has 1 too many active legislators
ar: District ar-upper-7 has 1 too many active legislators
ar: District ar-upper-32 has 1 too many active legislators
ar: District ar-upper-27 has 1 too many active legislators
ar: District ar-upper-22 has 1 too many active legislators
ar: District ar-lower-95 has 1 too many active legislators
ar: District ar-lower-89 has 1 too many active legislators
ar: District ar-lower-88 has 1 too many active legislators
ar: District ar-lower-7 has 1 too many active legislators
ar: District ar-lower-69 has 1 too many active legislators
ar: District ar-lower-68 has 1 too many active legislators
ar: District ar-lower-57 has 1 too many active legislators
ar: District ar-lower-54 has 1 too many active legislators
ar: District ar-lower-30 has 1 too many active legislators
ar: District ar-lower-3 has 1 too many active legislators
ar: District ar-lower-27 has 1 too many active legislators
ar: District ar-lower-25 has 2 too many active legislators
ar: District ar-lower-20 has 1 too many active legislators
ar: District ar-lower-14 has 1 too many active legislators

co: District co-lower-15 has 1 too many active legislators
co: District co-upper-33 has 1 too many active legislators

ct: District ct-upper-17 has 1 too many active legislators
ct: District ct-lower-97 has 1 too many active legislators

dc: District dc-upper-Ward 7 has 1 too many active legislators
dc: District dc-upper-At-Large has 3 too many active legislators

ky: District ky-lower-50 has 1 too many active legislators

ma: District ma-upper-Plymouth and Norfolk has 1 too many active legislators
ma: District ma-upper-Berkshire, Hampshire, Franklin and Hampden has 1 too many active legislators

md: District md-upper-40 has 1 too many active legislators
md: District md-upper-20 has 1 too many active legislators
md: District md-lower-46 has 1 too many active legislators
md: District md-lower-41 has 1 too many active legislators
md: District md-lower-40 has 1 too many active legislators
md: District md-lower-29C has 1 too many active legislators
md: District md-lower-20 has 1 too many active legislators

nj: District nj-lower-40 has 1 too many active legislators

pr: District pr-lower-29 has 1 too many active legislators

ri: District ri-upper-28 has 1 too many active legislators
ri: District ri-lower-5 has 1 too many active legislators

sc: District sc-upper-38 has 1 too many active legislators

sd: District sd-lower-8 has 1 too many active legislators

tn: District tn-upper-4 has 1 too many active legislators

ut: District ut-upper-20 has 1 too many active legislators
ut: District ut-upper-13 has 1 too many active legislators
ut: District ut-lower-70 has 1 too many active legislators
ut: District ut-lower-69 has 1 too many active legislators
ut: District ut-lower-6 has 1 too many active legislators
ut: District ut-lower-54 has 1 too many active legislators
ut: District ut-lower-53 has 1 too many active legislators
ut: District ut-lower-50 has 1 too many active legislators
ut: District ut-lower-43 has 1 too many active legislators
ut: District ut-lower-34 has 1 too many active legislators
ut: District ut-lower-31 has 1 too many active legislators
ut: District ut-lower-30 has 1 too many active legislators
ut: District ut-lower-3 has 1 too many active legislators
ut: District ut-lower-2 has 1 too many active legislators
ut: District ut-lower-14 has 1 too many active legislators

va: District va-upper-9 has 2 too many active legislators
va: District va-upper-5 has 1 too many active legislators
va: District va-upper-22 has 1 too many active legislators
va: District va-lower-93 has 1 too many active legislators
va: District va-lower-85 has 1 too many active legislators
va: District va-lower-77 has 1 too many active legislators

vt: District vt-lower-Windsor-4-2 has 1 too many active legislators
vt: District vt-lower-Caledonia-2 has 1 too many active legislators
vt: District vt-lower-Bennington-Rutland has 1 too many active legislators

wa: District wa-lower-12 has 1 too many active legislators

wv: District wv-upper-2 has 1 too many active legislators
wy: District wy-upper-8 has 1 too many active legislators
wy: District wy-upper-6 has 1 too many active legislators
wy: District wy-upper-4 has 1 too many active legislators
wy: District wy-upper-20 has 1 too many active legislators
wy: District wy-upper-14 has 1 too many active legislators
wy: District wy-upper-12 has 1 too many active legislators
wy: District wy-upper-10 has 1 too many active legislators
wy: District wy-lower-9 has 1 too many active legislators
wy: District wy-lower-6 has 1 too many active legislators
wy: District wy-lower-58 has 1 too many active legislators
wy: District wy-lower-57 has 1 too many active legislators
wy: District wy-lower-56 has 1 too many active legislators
wy: District wy-lower-51 has 1 too many active legislators
wy: District wy-lower-46 has 1 too many active legislators
wy: District wy-lower-41 has 1 too many active legislators
wy: District wy-lower-36 has 1 too many active legislators
wy: District wy-lower-35 has 1 too many active legislators
wy: District wy-lower-34 has 1 too many active legislators
wy: District wy-lower-32 has 1 too many active legislators
wy: District wy-lower-26 has 1 too many active legislators
wy: District wy-lower-24 has 1 too many active legislators
wy: District wy-lower-19 has 1 too many active legislators
wy: District wy-lower-18 has 1 too many active legislators
wy: District wy-lower-16 has 1 too many active legislators
wy: District wy-lower-14 has 1 too many active legislators
wy: District wy-lower-11 has 1 too many active legislators
```

- [x] AR
- [x] CO
- [x]  CT
- [x]  DC
- [x] KY
- [x] MA
- [x] MD
- [x] NJ 
- [x]  PR
- [x]  RI
- [x]  SC
- [x] SD
- [x] TN
- [x]  UT
- [x]  VA
- [x] VT,
- [x] WA
- [x] WY

**Comment by jamesturk** [comment]
AR, UT, WY look like bugs & should be investigated

The others look like in-session retirements, and should be a pretty easy sweep

**Comment by jmcarp** [comment]
> we do have a tool to let us know about these

For visibility, where does that tool live?

**Comment by estaub** [comment]
FWIW, I also looked for states with a suspiciously large number of seat vacancies, and found nothing. 👍

**Comment by rshorey** [comment]
CT resolved by #1413

**Comment by jamesturk** [comment]
after going through @samtregar's fixes all of these are resolved
